bin/main

```diff
@@ -1,2 +1,58 @@
 #!/usr/bin/env ruby
-puts 'Hello, world! '
+
+# get the player1 name
+p 'Hey player1, whats your name?'
+player1 = gets.chomp
+p "hello #{player1}, Please select a token (x / o)"
+# get the player1 token (x / o)
+player1_token = gets.chomp
+p "#{player1_token}, Nice choice!"
+
+# get the player2 name
+p 'Hey player2, whats your name?'
+player2 = gets.chomp
+# get the player2 token (x / o)
+player2_token = if player1_token == 'x'
+                  'o'
+                else 'x'
+                end
+p "hello #{player2}, your token is #{player2_token}"
+
+# create a loop to take each player move
+5.times do
```

Comment:
This is not OK, you should have a `game_over` boolean variable that tells when the game is over and iterate until this variable is `true`.